scaling_performance/optimizing_storage.adoc

```diff
@@ -86,8 +86,7 @@ The following table summarizes the recommended and configurable storage technolo
 | Yes
 | Configurable
 | Configurable
-| Configurable footnoteref:[metrics-warning,For metrics, it is an anti-pattern to use any shared storage and a single volume
```

Comment:
@jboxman can we actually leave that footnote but change it to: 

> For metrics, it is an anti-pattern to use any shared storage and a single volume (RWX).

I think that actually does represent that no one should ever configure RWX.

Reply:
Remove _By default, metrics deploys with one volume per Cassandra Replica._ this.